src/main/java/nextstep/subway/domain/exception/CannotDeleteSectionException.java

```diff
@@ -0,0 +1,16 @@
+package nextstep.subway.domain.exception;
+
+public class CannotDeleteSectionException extends RuntimeException {
+
+    private static final String MINIMUM_LIMIT_MESSAGE = "노선의 구간은 1개 이상 존재해야 합니다.";
+    private static final String NOT_EXISTS_MESSAGE = "노선에 없는 역은 삭제할 수 없습니다. [%s]";
+
+    public CannotDeleteSectionException() {
+        super(MINIMUM_LIMIT_MESSAGE);
+    }
+
+    public CannotDeleteSectionException(String stationName) {
+        super(String.format(NOT_EXISTS_MESSAGE, stationName));
+    }
```

Comment:
단순히 생성자의 인자만 보고서는 두 생성자 중에 어느 것을 사용해야하는지가 명확하지 않은 것 같습니다.
static method 등으로 명시적으로 다른 컨텍스트다임을 나타내는 네이밍을 부여해주어도 좋다는 생각입니다 :)

대강 아래와 같은 코드가 나올 수 있을 것 같네요!

```suggestion
    private static final String MINIMUM_LIMIT_MESSAGE = "노선의 구간은 1개 이상 존재해야 합니다.";
    private static final String NOT_EXISTS_MESSAGE = "노선에 없는 역은 삭제할 수 없습니다. [%s]";

    private CannotDeleteSectionException() {
        super(MINIMUM_LIMIT_MESSAGE);
    }

    private CannotDeleteSectionException(String stationName) {
        super(String.format(NOT_EXISTS_MESSAGE, stationName));
    }
    
    public static CannotDeleteSectionException minimumLimit() {
        return new CannotDeleteSectionException();
    }
    
    public static CannotDeleteSectionException notExists(String stationName) {
        return new CannotDeleteSectionException(stationName);
    }
```

Reply:
예시 코드 😲👍
저의 부족한 응용력에 반성하게 되는 피드백입니다 ㅎㅎㅎ
좋은 코드 보여주셔서 감사합니다 !